Changes.md

```diff
@@ -59,6 +59,7 @@ API
 - ValuePlug
   - Improved interactive performance by not clearing the entire hash cache every time a plug is dirtied. Beware : this can reveal subtle bugs in `DependencyNode::affects()` implementations, causing hashes to be reused if a plug has not been dirtied appropriately. These bugs may previously have gone unnoticed but will now need fixing as a matter of urgency. The GAFFER_CLEAR_HASHCACHE_ON_DIRTY environment variable may be used to enable legacy behaviour in the interim.
   - Added `clearHashCache()` static method.
+  - Added `resetDefault()` method.
```

Comment:
We need a big "References with promoted spreadsheets saved using Gaffer 0.58 are not compatible with previous versions of Gaffer" warning.

![image](https://user-images.githubusercontent.com/896779/88678623-93dcd480-d0e6-11ea-83e8-3f0adb8be2f7.png)

Reply:
Very good point - added in f450042. Although I suppose all that has really changed is we've replaced a silent failure with a loud one.